Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -423,6 +429,9 @@ void NetworkSession::handlePrivateClickMeasurementConversion(WebCore::PCM::Attri
         return;
     }
 
+    if (m_sessionID.isEphemeral())
```

Comment:
Is this just a safety measure? It feel strange to have below all of the handling of ephemeral measurement. Could it instead be an early check that if it's an ephemeral session and there is no pending ephemeral measurement, we return?

Reply:
Hm...so I think we still want to early return here, otherwise we might end up with some confusing console messages about accepted triggering events, even though the ephemeral store would effectively ignore these attribution events. I'll move it up in this function, in the way you described.